cristin-person/src/test/java/no/unit/nva/cristin/person/handler/PersonKarateRunner.java

```diff
@@ -6,6 +6,7 @@
 @Tag("karateTest")
 public class PersonKarateRunner {
 
+    @Tag("karateTest")
```

Comment:
Why do you need this here as well?

Reply:
No need for that, was a gradle problem...